input/fsh/examples/encounter.fsh

```diff
@@ -26,10 +26,10 @@ Description: "An example ManaakiNgaTahi Encounter"
 * length = 7 'd' "days"
 * reasonCode = http://snomed.info/sct#270427003 "patient-initiated encounter"
 * reasonCode.text = "Patient Initiated Encounter"
-* diagnosis.condition = Reference(ConditionExample)
```

Comment:
Any reason for removing the condition reference form the encounter?

Reply:
ditto above